### PR TITLE
`General`: Provide missing German translation for the about us page

### PR DESCRIPTION
**Pull request by Dominik-Weinzierl**
#### General

- [ ] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context

Closes: https://github.com/ls1intum/Artemis/issues/5359

### Description

Added additional translation.

### Steps for Testing

Prerequisites:
- 1 User

1. Log in to Artemis
2. Navigate to the About us page
3. Switch to German
4. Verify the correct translation

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### Screenshots

image

### CODE REVIEW

**Review by Strohgelaender — APPROVED** [review]
code 👍

**Review by Marvin-Vitcu — COMMENTED** [review]
Tested on TS2. The translation for Project Manager worked.
However, the translation for website is still missing. (website = Webseite)

I would also use another word than website. Maybe something like "profile page" / "profile website". But just "website" confuses me as a user, as I do not know where this website is going.
image

**Review by luisa-we — COMMENTED** [review]
Tested on TS2. The translation for Project Manager didn't work for me:

Bildschirmfoto 2022-07-11 um 14 37 46

**Comment by Dominik-Weinzierl** [comment]
@Marvin-Vitcu @luisa-we Thanks for the suggestion! Updated :+1:

**Review by jschmautzer — DISMISSED** [review]
tested on ts2, worked as described

**Review by stefanwaldhauser — APPROVED** [review]

**Review by alaraozdenler — APPROVED** [review]
tested on ts2, looks good

File: src/main/webapp/i18n/de/aboutUs.json [hunk 1]
```diff
@@ -62,6 +62,8 @@
             "text": "<b>Open-source:</b> Kostenlos zu verwenden mit einer großen Gemeinschaft und vielen aktiven Entwicklern."
         },
         "getInTouch": "Kontakt",
-        "contributors": "Mitwirkende"
+        "contributors": "Mitwirkende",
+        "projectManager": "Projektmanager",
```
Comment (krusche) [inline src/main/webapp/i18n/de/aboutUs.json:66]:
```suggestion
        "projectManager": "Projekt Manager",
```

**Pull request merged**